### PR TITLE
Changes to iterm relax and adding direct FF

### PR DESCRIPTION
**Pull request by Quick-Flash**
Changed iterm relax so that rather than an off/on case it will grow and disappear. Also added some amount of iterm relax effect to yaw, which should help with some iterm windup. Yaw has its own iterm relax values.

Added direct FF and made setpoint always calculated. This means that in angle mode dterm setpoint weight has effect in angle mode as well as direct FF. Direct FF scales with your setpoint giving a push based on how large your setpoint is. This works very well on yaw as yaw always needs some amount of push, which this can approximate. It also works well in angle mode as it allows angle mode to directly effect the rate pidsum (as now angle mode creates a setpoint) and adds an extra "kick" in angle mode that is different than what angle Pterm does. 

TODO: recode the direct ff so that it is much easier for users to tune and use. Just need to figure out the proper way that it should be done, open for help with that if this feature proves desirable.

Would love to hear users experience with both these features.

### CODE REVIEW

**Comment by bkleiner** [comment]
gotta figure out a good way to build and attach hexes to PRs so we can have people test it

File: src/main/config/config.h [hunk 1]
```diff
@@ -1,6 +1,8 @@
 
+#define KDFF_ROLL 0.0f // TODO, move and allow tunable, just unsure how atm
+#define KDFF_PITCH 0.0f
+#define KDFF_YAW 0.000286479f
```
[the remark this answers is not in the document]
Comment (Quick-Flash) [reply src/main/config/config.h:4]: Something a user would want to tweak.

File: src/main/config/config.h [hunk 2]
```diff
@@ -257,8 +259,12 @@
 
 //**************I-term relax.  Removes roll and pitch bounce back after flips
 #define I_TERM_RELAX
-//#define RELAX_FACTOR 10
-//#define RELAX_FREQUENCY_HZ 20
+#define RELAX_FACTOR 35 * 0.01745329251 // convert to rad
```
[the remark this answers is not in the document]
Comment (Quick-Flash) [reply src/main/config/config.h:262]: Can you put a function into a define? If so we could just define it as converted to rad/s

[the remark this answers is not in the document]
Comment (bkleiner) [reply src/main/config/config.h:262]: in theory yes, but really i would like the config.h stay clear of such constructs.

**Comment by bkleiner** [comment]
the iterm portion of this PR has been fixed up and separated out to https://github.com/BossHobby/QUICKSILVER/pull/57 and is currently undergoing testing.

**Comment by bkleiner** [comment]
Closing as outdated. If we were to got forward, we can pull changes manually.

**Pull request closed without merging**